client/my-sites/plans/main.jsx

```diff
@@ -31,7 +31,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
ok, wait, I didn't realise that displayJetpackPlans was a synthetic prop we are assigning in the connector of plans/main.jsx... I was assuming we were passing it from the controller. Ok, yeah, being like this, I have no further complains :)

![](http://www.vomzi.com/wp-content/uploads/2016/02/best-im-sorry-gif-668.gif)